scripts/microbe_stage/microbe_operations.as

```diff
@@ -176,6 +176,10 @@ void respawnPlayer(CellStageWorld@ world){
     setRandomBiome(world);
     cast<MicrobeStageHudSystem>(world.GetScriptSystem("MicrobeStageHudSystem")).
         suicideButtonreset();
+		
+	//Decrease the population by 10
+	auto localPlayer = MicrobeOperations::getSpeciesComponent(world, "Default");
```

Comment:
`localPlayer` should be renamed to `playerSpecies` as this isn't an object representing the player rather it represents the player's species.